github/ci/prow-deploy/files/jobs/RHsyseng/rhcos-slb/rhcos-slb-presubmits.yaml

```diff
@@ -0,0 +1,35 @@
+---
+presubmits:
+  RHsyseng/rhcos-slb:
+    - name: coreos-assembler-ci-network-tests-main
```

Comment:
the presubmit jobs for PRs usually use the "pull-" prefix to denote that they are running on pull-request events not periodics.

Reply:
totally.